src/event-notion/index.ts

```diff
@@ -696,21 +696,27 @@ export const pingForTAPandCSIDeadlines = async (
  *
  * Every event in any CSE building room that is not CSE B225 ("The Fishbowl") requires keys.
  */
+const keyPingDays = 4;
```

Comment:
Do we want to add a comment explaining why this is 4?

Reply:
For sure — just added this + added more description to the PR